vispy/app/base.py

```diff
@@ -184,12 +184,15 @@ def _vispy_mouse_move(self, **kwargs):
 
     def _vispy_mouse_release(self, **kwargs):
         # default method for delivering mouse release events to the canvas
+
         kwargs.update(self._vispy_mouse_data)
         ev = self._vispy_canvas.events.mouse_release(**kwargs)
-        if ev.button == self._vispy_mouse_data['press_event'].button:
+        if (self._vispy_mouse_data['press_event'] 
+            and  self._vispy_mouse_data['press_event'].button == ev.button) :
```

Comment:
I think I got it now.  Thanks for your patience...

On Wed, Mar 25, 2015 at 10:49 AM, Eric Larson <notifications@github.com>
wrote:

> In vispy/app/base.py
> <https://github.com/vispy/vispy/pull/773#discussion_r27146801>:
>
> >          kwargs.update(self._vispy_mouse_data)
> >          ev = self._vispy_canvas.events.mouse_release(**kwargs)
> > -        if ev.button == self._vispy_mouse_data['press_event'].button:
> > +        if (self._vispy_mouse_data['press_event']
> > +            and  self._vispy_mouse_data['press_event'].button == ev.button) :
>
> Did you run a PEP8 checker? I see two or three new errors on this line...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vispy/vispy/pull/773/files#r27146801>.
>